dev-libs/openssl/openssl3-3.0.13.recipe

```diff
@@ -21,7 +21,7 @@ CHECKSUM_SHA256="88525753f79d3bec27d2fa7c66aa0b92b3aa9498dafd93d7cfa4b3780cdae31
 SOURCE_DIR="openssl-$portVersion"
 PATCHES="openssl3-$portVersion.patchset"
 
-ARCHITECTURES="?all"
+ARCHITECTURES="all"
 SECONDARY_ARCHITECTURES="?x86"
```

Comment:
Why not enabling for x86?

Reply:
I enabled only what I could test. I did not boot my 32bit system this weekend. I can make sure everything works there as well if no one else does it before me.

Reply:
I tested the x86 and x86_gcc2 builds, they seem OK with the master patchset.

Reply:
Builds went fine here for 32bit too, even building the haiku anyboot image went well on 32bit (need to recheck with latest changes), in qemu everything seems to be working fine.